liquidctl/driver/nzxt_smart_device.py

```diff
@@ -121,39 +159,55 @@
 _MAX_DUTY = 100
 _READ_ENDPOINT = 0x81
 _READ_LENGTH = 21
+_READ_LENGTH_V2 = 60     # for Smart Device V2
 _WRITE_ENDPOINT = 0x1
 _WRITE_LENGTH = 65
 
-
 class NzxtSmartDeviceDriver(UsbHidDriver):
     """USB driver for the NZXT Smart Device and Grid+ V3."""
 
+    DEVICE_SMARTDEV_V1 = 'Smart Device V1'
+    DEVICE_GRID_V3  = 'Grid V3'
+    DEVICE_SMARTDEV_V2 = 'Smart Device V2'
+
     SUPPORTED_DEVICES = [
         (0x1e71, 0x1714, None, 'NZXT Smart Device', {
             'speed_channel_count': 3,
-            'color_channel_count': 1
+            'color_channel_count': 1,
+            'device_type': DEVICE_SMARTDEV_V1
         }),
         (0x1e71, 0x1711, None, 'NZXT Grid+ V3 (experimental)', {
             'speed_channel_count': 6,
-            'color_channel_count': 0
+            'color_channel_count': 0,
+            'device_type': DEVICE_GRID_V3
+        }),
+        (0x1e71, 0x2006, None, 'NZXT Smart Device V2 (experimental)', {
+            'speed_channel_count': 3,
+            'color_channel_count': 2,
+            'device_type': DEVICE_SMARTDEV_V2
         }),
     ]
 
-    def __init__(self, device, description, speed_channel_count, color_channel_count, **kwargs):
+    def __init__(self, device, description, speed_channel_count, color_channel_count, device_type=DEVICE_SMARTDEV_V1,  **kwargs):
         """Instantiate a driver with a device handle."""
         super().__init__(device, description)
         self._speed_channels = {'fan{}'.format(i + 1): (i, _MIN_DUTY, _MAX_DUTY)
                                 for i in range(speed_channel_count)}
-        self._color_channels = {'sync': (0)} if color_channel_count else {}
+        self._color_channels = {'sync{}'.format(i + 1): (i)
+                                for i in range(color_channel_count)}
```

Comment:
I've changed it to `led1` and `led2`. This may not be ideal, but it's the terminology NZXT uses on its website.

>        self._color_channels = {'led{}'.format(i + 1): (i)
>                                for i in range(color_channel_count)}